js/search.js

```diff
@@ -73,7 +77,7 @@
 			}
 
 			this.activeSearch = searchText;
-			this.$element.addClass('searched');
+			this.$element.addClass('searched pending');
```

Comment:
Would it be better to use a variable on `this` instead of storing this state in a class name?